### PR TITLE
fix(tutorial): README didn't reflected actual code

### PR DESCRIPTION
**Pull request by EladBezalel**
- added toolbar tools wrapper
- fixed README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,25 +171,27 @@ export default angular.module( "starter-app", [ 'ngMaterial' ] )
...
+* Use the `md-whiteframe-2dp` to add a shadow the the sidenav
 
 ```html
-  <body ng-app="starterApp" layout="column">
+  <body layout="column">
 
 	<!-- Container #1 (see wireframe) -->
     <md-toolbar layout="row" >
-      <h1>Angular Material - Starter App</h1>
+      <div class="md-toolbar-tools">
```
Comment (topherfangio) [inline README.md]: We should definitely bring this up during the workshop about what it does. We might also want to update the toolbar docs to talk about it.

File: app/tutorial_5.html [hunk 2]
```diff
@@ -16,7 +16,9 @@
   <body ng-cloak layout="column" ng-controller="UsersController as ul" ng-cloak>
 
     <md-toolbar layout="row">
-      <h3>Angular Material - Starter App</h3>
+      <div class="md-toolbar-tools">
```
Comment (ThomasBurleson) [inline app/tutorial_5.html]: why not `<md-toolbar layout="row" class="md-toolbar-tools">`

File: app/boot.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import angular from 'angular';
-import App from './src/App';
+import App from './src/app';
```
Comment (ThomasBurleson) [inline app/boot.js]: `App.js` is uppercase.

**Comment by ThomasBurleson** [comment]
Closed with merge SHA #ff78083ae63018f6f454

**Pull request closed without merging**